src/shaders/symbol_icon.vertex.glsl

```diff
@@ -72,19 +75,24 @@ void main() {
     highp float symbol_rotation = 0.0;
     if (u_rotate_symbol) {
         // See comments in symbol_sdf.vertex
-        vec4 offsetProjectedPoint = u_matrix * vec4(a_pos + vec2(1, 0), h, 1);
+        vec4 offsetProjectedPoint = u_matrix * vec4(a_pos + vec2(1, 0), anchorZ, 1);
 
         vec2 a = projectedPoint.xy / projectedPoint.w;
         vec2 b = offsetProjectedPoint.xy / offsetProjectedPoint.w;
 
         symbol_rotation = atan((b.y - a.y) / u_aspect_ratio, b.x - a.x);
     }
 
+#ifdef PROJECTED_POS_ON_VIEWPORT
```

Comment:
Note: `PROJECTED_POS_ON_VIEWPORT` is not used at this point. It is required later on to properly handle labels that have pitch alignment set to "map".